.env.default

```diff
@@ -0,0 +1,9 @@
+BERLN_API_HOST=0.0.0.0
+BERLN_API_PORT=28900
+
+BERLIN_API_NAMESPACE=dp_nlp_berlin_api
```

Comment:
This is different to `BERLIN_API_LOGGING_NAMESPACE` and `LOGGING_NAMESPACE` in other files.

Reply:
Not sure what you mean, BERLIN_API is just the prefix for dynaconf. 
So you set what the prefix will be and that instance of dynaconf will take only env variables that start with BERLIN_API
and you get them by querying for the env var without the prefix e.g. LOGGING_NAMESPACE
if that's what you meant tho

Reply:
so here it is "NAMESPACE"
elsewhere it is "LOGGING_NAMESPACE"